frontend/dockerfile/dockerignore/dockerignore_deprecated.go

```diff
@@ -0,0 +1,14 @@
+package dockerignore
+
+import (
+	"io"
+
+	"github.com/moby/patternmatcher/ignorefile"
+)
+
+// ReadAll is an alias for [ignorefile.ReadAll].
+//
+// Deprecated: use [ignorefile.ReadAll] instead.
```

Comment:
I don't think we need this deprecation file

Reply:
Without it, external consumers won't discover that it was moved, and will be caught off-guard when the alias is removed, and now need to search "where the heck" it went.